src/presentation/pages/Note.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    Note id {{ note.id }}
+  </div>
+  <div>
+    Note title: {{ note.title }}
+  </div>
+  <div>
+    Note content: {{ note.content }}
+  </div>
+</template>
+
+<script lang="ts">
+import NoteService from "../../domain/note.service";
+import useNote from "../../application/services/useNote";
+
+export default {
+  name: 'Note',
+  props: {
+    noteService: {
+      type: NoteService,
+      required: true,
+    },
+    id: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
```

Comment:
let's use `<script setup>` syntax instead, it's more readable

Reply:
done